content/en/docs/components/pipelines/sdk/v2/build-pipeline.md

```diff
@@ -0,0 +1,520 @@
++++
+title = "Build a Pipeline"
+description = "A tutorial on using Pipelines SDK v2 to orchestrate your ML workflow as a pipeline"
+weight = 30
++++
+
+<!--
+AUTOGENERATED FROM content/en/docs/components/pipelines/sdk/v2/build-pipeline.ipynb
+PLEASE UPDATE THE JUPYTER NOTEBOOK AND REGENERATE THIS FILE USING scripts/nb_to_md.py.-->
+
+<style>
+.notebook-links {display: flex; margin: 1em 0;}
+.notebook-links a {padding: .75em; margin-right: .75em; font-weight: bold;}
+a.colab-link {
+padding-left: 3.25em;
+background-image: url(/docs/images/logos/colab.ico);
+background-repeat: no-repeat;
+background-size: contain;
+}
+a.github-link {
+padding-left: 2.75em;
+background-image: url(/docs/images/logos/github.png);
+background-repeat: no-repeat;
+background-size: auto 75%;
+background-position: left center;
+}
+</style>
+<div class="notebook-links">
+<a class="colab-link" href="https://colab.research.google.com/github/kubeflow/website/blob/master/content/en/docs/components/pipelines/sdk/v2/build-pipeline.ipynb">Run in Google Colab</a>
+<a class="github-link" href="https://github.com/kubeflow/website/blob/master/content/en/docs/components/pipelines/sdk/v2/build-pipeline.ipynb">View source on GitHub</a>
+</div>
+
+
+
+A Kubeflow pipeline is a portable and scalable definition of a machine learning
+(ML) workflow. Each step in your ML workflow, such as preparing data or
+training a model, is an instance of a pipeline component. This document
+provides an overview of pipeline concepts and best practices, and instructions
+describing how to build an ML pipeline.
+
+**Note:** This guide demonstrates how to build pipelines using the Pipelines SDK v2.
+Currently, Kubeflow Pipelines v2 is in development. You can use this guide to start
+building and running pipelines that are compatible with the Pipelines SDK v2.
+
+[Learn more about Pipelines SDK v2][kfpv2].
+
+[kfpv2]: https://www.kubeflow.org/docs/components/pipelines/sdk/v2/v2-compatibility
+
+## Before you begin
+
+1.  Run the following command to install the Kubeflow Pipelines SDK v1.6 or higher.
+    If you run this command in a Jupyter notebook, restart the kernel after
+    installing the SDK. 
+
+
+```python
```

Comment:
assuming this will be fixed later to `bash`

Reply:
Yes, this is an issue in the notebook to MD script. I'll update the Markdown files once the script is fixed.